python/cudf/cudf/_libxx/includes/hash.pxd

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cudf._libxx.lib cimport *
+
+cdef extern from "cudf/hashing.hpp" namespace "cudf" nogil:
+    cdef pair[unique_ptr[table], vector[size_type]] \
+        hash_partition "cudf::hash_partition" (
+        const table_view& input,
+        const vector[size_type]& columns_to_hash,
```

Comment:
Did you find that using the `const` and `&` changed the generated cpp for the better?

Reply:
I saw zero difference in the generated cpp... It made me feel a little better matching the real C++ signature though.